ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -102,6 +104,8 @@ const getDescription = (
         return t('confirmTitleDescApproveTransaction');
       }
       return t('confirmTitleDescERC20ApproveTransaction');
+    case TransactionType.tokenMethodIncreaseAllowance:
+      return t('confirmTitleDescPermitSignature');
```

Comment:
[suggestion] we can update these i18n keys since they are now being extended beyond signatures 

maybe
confirmTitleDescPermitSignature -> confirmTitleDescPermitTokens